### ISSUE TITLE
Discussion about screen rotation modes

### ISSUE THREAD

**Issue by lightonflux** [koreader/koreader#1338]
I really like that ko**reader** can rotate the screen, but there are some things i find very annoying.

 * Only the reader supports it ATM
 * It is enable per document

I would prefer it when the screen rotation would be "system wide" and depend on the system setting not the document setting. 

I don't know any usecase in which screen rotation on a per document basis make sense, but i am sure other people might need it. 

In other words: a mode that would mirror the behaviour of smartphones and similar devices would be a big help.
